lib/handle-schedule.ts

```diff
@@ -95,15 +99,39 @@ export default async function handleSchedule(): Promise<void> {
       });
       core.info(`${pullRequest.html_url} merged`);
     } catch (error) {
-      const { data } = await createComment(
+      const previousComment = await getPreviousComment(
         octokit,
         pullRequest.number,
-        generateBody(
-          `Scheduled merge failed: ${(error as Error).message}`,
-          "error"
-        )
+        "fail"
       );
-      core.info(`Comment created: ${data.html_url}`);
+      const commentBody = generateBody(
+        `Scheduled merge failed: ${
+          (error as Error).message
+        }\nIn order to let the automerge-automation try again, the label "${automergeFailLabel}" should be removed.`,
+        "error",
+        "fail"
+      );
+      if (previousComment) {
+        const { data } = await updateComment(
+          octokit,
+          previousComment.id,
+          commentBody
+        );
+        core.info(`Comment updated: ${data.html_url}`);
+      } else {
+        const { data } = await createComment(
+          octokit,
+          pullRequest.number,
+          commentBody
+        );
+        core.info(`Comment created: ${data.html_url}`);
+      }
+      await octokit.rest.issues.addLabels({
+        ...github.context.repo,
+        issue_number: pullRequest.number,
+        labels: [automergeFailLabel],
+      });
```

Comment:
I think a label gets created automatically if it doesn't exist in the repository yet, but can you double check?